hs/SvgParsing/SVGGenerator.hs

```diff
@@ -188,4 +190,11 @@ convertEllipseToSVG ellipse = S.g ! A.id_ (stringValue (shapeId ellipse))
                                                 ! A.rx (stringValue $ show $ fromRational $ shapeWidth ellipse / 2)
                                                 ! A.ry (stringValue $ show $ fromRational $ shapeHeight ellipse / 2)
                                                 ! A.style "stroke:#000000;fill:none;"
-                                      concatSVG $ map (convertTextToSVG False True False) (shapeText ellipse)
\ No newline at end of file
+                                      concatSVG $ map (convertTextToSVG False True False) (shapeText ellipse)
+
+getTextStyle :: Bool -> Bool -> Bool -> String
+getTextStyle isHybrid isBool isRegion
```

Comment:
As in 
```
getTextStyle :: Bool -> Bool -> Bool -> String
getTextStyle True True False = hybridTextStyle
getTextStyle False True False = ellipseTextStyle
getTextStyle False False True = regionTextStyle
getTextStyle _ _ _ = ""
```
?

Is there a better way we can approach separating texts that are in bools vs rects vs regions?